api/specs/webserver/v0/components/schemas/project-v0.0.1-converted.yaml

```diff
@@ -3,18 +3,18 @@ description: Description of a simcore project
 type: object
 additionalProperties: false
 required:
-  - projectUuid
+  - uuid
   - name
   - description
   - notes
-  - owner
+  - prjOwner
```

Comment:
I wonder why this change. You remove prefix project to ``uuid`` but adds it to ``owner``.
Is ``owner`` a reserved word?

Reply:
Correct, "owner" is kind of reserved word in the qx-compiler we are using at the moment. I think that the latest compiler already avoids using it though.